<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogAttrDefSet</class>
 <widget class="QDialog" name="DialogAttrDefSet">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>180</width>
    <height>199</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>属性文字设置</string>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>34</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>对正：</string>
   </property>
  </widget>
  <widget class="QComboBox" name="CbAlignment">
   <property name="geometry">
    <rect>
     <x>48</x>
     <y>30</y>
     <width>120</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>左对齐</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>居中对齐</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>右对齐</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>62</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>字高：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="EdHeight">
   <property name="geometry">
    <rect>
     <x>48</x>
     <y>57</y>
     <width>120</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>2.5</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>字宽：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="EdWidth">
   <property name="geometry">
    <rect>
     <x>48</x>
     <y>85</y>
     <width>120</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>0.8</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>118</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>旋转：</string>
   </property>
  </widget>
  <widget class="QComboBox" name="CbRotation">
   <property name="geometry">
    <rect>
     <x>48</x>
     <y>113</y>
     <width>120</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>0</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>90</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>180</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>270</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="BtnOk">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>60</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>确 定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BtnCancel">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>170</y>
     <width>60</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>取 消</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="EdTag">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>5</y>
     <width>158</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>146</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>颜色：</string>
   </property>
  </widget>
  <widget class="QComboBox" name="CbColour">
   <property name="geometry">
    <rect>
     <x>48</x>
     <y>141</y>
     <width>120</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>白色</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>蓝色</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>青色</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>绿色</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>洋红色</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>红色</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>黄色</string>
    </property>
   </item>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
